File: commands/utility/calculate.js
const math = require ('mathjs')

module.exports = {
    name: 'calculate',
    aliases: ['cal'],
    permissions: [],
    cooldown: 0,
    async execute(client, message, cmd, args, Discord) {

        if (!args[0]) return message.channel.send({ embed: { color: `#00f2ff`, description: 'Please provide a question' } });

        let resp;

        try {
            resp = math.evaluate(args.join(" "))
        } catch (e) {
            return message.channel.send({ embed: { color: `#00f2ff`, description: 'Please provide a **valid** question' } })
        }

        const embed = new Discord.MessageEmbed()
            .setColor('#00f2ff')
            .setTitle('Calculator')
            .addField('Question', `\`\`\`css\n${args.join(' ')}\`\`\``)
            .addField('Answer', `\`\`\`css\n${resp}\`\`\``)

        message.channel.send(embed);
    }
}

File: commands/Fun/calculate.py
from setting import *

@client.command(aliases=['calc'])
async def calculate(ctx, maths: str):

    # Embeding
    embed = discord.Embed(title="Calculator", color=0x00f2ff)
    embed.add_field(name="Question",
                    value=f"```css\n{maths}```", inline=False)
    
    # Answer
    embed.add_field(
        name="Answer", value=f"```css\n{str(eval(maths))}```", inline=False)
    await ctx.send(embed=embed)

# Error(s)
@calculate.error
async def calculate_error(ctx, error):
    
    # Missing Argument(s)
    if isinstance(error, commands.MissingRequiredArgument):
        embed = discord.Embed(
            title="Opprator error", description=f"Please give opprator and number", color=0x00f2ff)
        embed.add_field(name=f"Example", value=f"{prefix}calculate 5 + 5")
        await ctx.send(embed=embed)
    
    # Number error
    elif isinstance(error, commands.CommandInvokeError):
        embed = discord.Embed(
            title="Number error", description=f"Please give a valid number", color=0x00f2ff)
        embed.add_field(name=f"Example", value=f"{prefix}calculate 5 + 5")
        await ctx.send(embed=embed)
        
    # Unknown error
    else:
        await unknown_error(ctx, error)

